chartfx-chart/src/test/java/de/gsi/chart/plugins/ScreenshotTest.java

```diff
@@ -0,0 +1,146 @@
+package de.gsi.chart.plugins;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import javax.imageio.ImageIO;
+
+import javafx.embed.swing.SwingFXUtils;
+import javafx.scene.Scene;
+import javafx.scene.image.Image;
+import javafx.scene.image.WritableImage;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.DataFormat;
+import javafx.scene.layout.FlowPane;
+import javafx.stage.Stage;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.testfx.api.FxAssert;
+import org.testfx.api.FxRobot;
+import org.testfx.framework.junit5.ApplicationExtension;
+import org.testfx.framework.junit5.Start;
+
+import de.gsi.chart.XYChart;
+import de.gsi.dataset.spi.DoubleDataSet;
+import de.gsi.dataset.testdata.spi.CosineFunction;
+
+/**
+ * Test the screenshot plugin
+ * 
+ * @author Alexander Krimm
+ */
+@ExtendWith(ApplicationExtension.class)
+class ScreenshotTest {
+    private Scene scene;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'scene' could be replaced by a local variable.](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5084368)